golang/cosmos/x/vbank/vbank_test.go

```diff
@@ -730,16 +746,32 @@ func Test_EndBlock_Rewards(t *testing.T) {
 	}
 }
 
-type mockAccount struct{}
+type mockAccount struct{
+	accs map[string]authtypes.AccountI
+}
 
 func (ma mockAccount) GetModuleAddress(name string) sdk.AccAddress {
 	return sdk.AccAddress(name)
 }
 
+func (ma mockAccount) GetAccount(ctx sdk.Context, addr sdk.AccAddress) authtypes.AccountI {
+	fmt.Printf("GetAccount %s\n", addr.String())
+	return ma.accs[addr.String()]
+}
+
 func Test_Module_Account(t *testing.T) {
-	account := &mockAccount{}
-	keeper, ctx := makeTestKit(account, nil)
-	ch := NewPortHandler(AppModule{}, keeper)
+	acct := &mockAccount{
+		accs: map[string]authtypes.AccountI{
+			"cosmos1we3xzmnt9aex2um9wfmx2em0pd0": &authtypes.ModuleAccount{
```

Comment:
Consider making the address a named constant for easier visual comparison within the test.